src/sarif_converter.ts

```diff
@@ -209,7 +209,7 @@ function getResults(allIssues: ISarifSuggestion[]): Result[] {
 
     if (result.message.text) {
```

Comment:
question: same question as in [previous PR](https://github.com/snyk/octopus/pull/911#discussion_r651865003), is the predicate correct?